portal/templates/utilisation-report-service/record-correction/_macros/correction-request-details-table.njk

```diff
@@ -0,0 +1,31 @@
+{% macro render(params) %}
+  {% set details = params.details %}
+
+  <table class="govuk-table" data-cy="correction-request-details-table">
+    <caption class="govuk-table__caption govuk-table__caption--m">Current record</caption>
+    <thead class="govuk-table__head">
+      <tr class="govuk-table__row">
+        <th scope="col" class="govuk-table__header">Facility ID</th>
+        <th scope="col" class="govuk-table__header">Exporter</th>
+        <th scope="col" class="govuk-table__header">Reported fees</th>
+        <th scope="col" class="govuk-table__header" data-cy="correction-request-details-table-header--error-type">
+          {% if details.reasons.length > 1 %}
+            Error types
+          {% else %}
+            Error type
+          {% endif %}
+        </th>
+        <th scope="col" class="govuk-table__header">Error summary</th>
+      </tr>
+    </thead>
+    <tbody class="govuk-table__body">
+      <tr class="govuk-table__row">
+        <td class="govuk-table__cell" data-cy="correction-request-details-table--facility-id">{{ details.facilityId }}</td>
+        <td class="govuk-table__cell" data-cy="correction-request-details-table--exporter">{{ details.exporter }}</td>
+        <td class="govuk-table__cell" data-cy="correction-request-details-table--reported-fees">{{ details.reportedFees }}</td>
+        <td class="govuk-table__cell" data-cy="correction-request-details-table--formatted-reasons">{{ details.formattedReasons }}</td>
+        <td class="govuk-table__cell" data-cy="correction-request-details-table--additional-info">{{ details.additionalInfo }}</td>
```

Comment:
Line break handling? You can nab a specific commit from my branch (pending corrections) to have the nunjucks filter I added to TFM in portal

Reply:
Ah yes, thank you, cherry picked that commit (`3a228acced67307ac818ce8b7a65c57bc833db7e`) and updated this macro